metadata-checker/popup/metadata.js

```diff
@@ -0,0 +1,138 @@
+function onMetaData(metaData) {
+  metaData = metaData[0];
+
+  var shown = false;
+
+  for (prop in metaData) {
+    value = metaData[prop];
+    var section = null;
+    if (!prop.match(/^og:/) && !prop.match(/^twitter:/) && !prop.match(/^fb:/))
+      continue;
+    if (!shown) {
+      document.getElementById("metadata").setAttribute("class", "");
+      document.getElementById("nothing-to-see").setAttribute("class", "hidden");
+    }
+    let ul = document.getElementById("metadata-list");
```

Comment:
You sometimes use `let` and sometimes `var`, without an obvious reason (not obvious to me anyway :). Try to be consistent throughout the extension (some people seem to still use `var` everywhere, which is fine IMO, but if you use other forms, please use them consistently).

Reply:
I actually always used var up to now, but somewhere else I saw let and var being used and I somehow got the impression that let was kind of a "const" or something, but I didn't really ever looked into the differences. Now [I get it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) :D